### PR TITLE
docs(readme): add MagicLight WebBLE integration

### PR DESCRIPTION
**Pull request by manekinekko**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -247,6 +247,7 @@ This section details projects that have integrated Lighthouse. If you're working
 * **[lighthouse-mocha-example](https://github.com/justinribeiro/lighthouse-mocha-example)** - gather performance metrics via Lighthouse and tests them in Mocha
 * **[pwmetrics](https://github.com/paulirish/pwmetrics/)** - gather performance metrics
 * **[lighthouse-hue](https://github.com/ebidel/lighthouse-hue)** - set the color of Philips Hue lights based on a Lighthouse score
+* **[lighthous-magic-light](https://github.com/manekinekko/lighthouse-magic-light)** set the color of the MagicLight Bluetooth Smart Light Bulb based on Lighthouse score
```
Comment (paulirish) [inline readme.md]: wanna add 3 spaces at the start to indent this under lighthouse-hue since they're buddies?

**Comment by manekinekko** [comment]
Sure. Will do that.

**Comment by brendankenny** [comment]
Can you also update the README on the project? Going in there users might miss the updates and only see the references to Phillips Hue in the readme.

**Comment by manekinekko** [comment]
@paulirish @brendankenny I update my project on master. Everyhting should be OK now. lmk.

**Pull request merged**